CrowsZA/functions/fn_zeusRegister.sqf

```diff
@@ -49,6 +49,7 @@ private _moduleList = [
     ["Resupply Player Loadouts",{_this call crowsZA_fnc_resupplyPlayerLoadouts}, "\CrowsZA\data\resupplyplayerloadout.paa"],
     ["Remove Radio/Bino",{_this call crowsZA_fnc_removeRadioBino}, "\a3\Ui_f\data\GUI\Cfg\CommunicationMenu\call_ca.paa"],
     ["Strip Explosives",{_this call crowsZA_fnc_stripExplosivesZeus}, "\a3\ui_f\data\igui\cfg\simpletasks\types\destroy_ca.paa"],
+    ["Surrender Chance",{_this call crowsZA_fnc_surrenderChanceZeus}, "\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa"],
```

Comment:
Does case-sensitivity matter in path? I see the line above is referencing "igui" while this one is using "IGUI"? Same with "cfg" and "Cfg". Not sure if it matters?

Reply:
Doesn't seem to matter, but have updated for consistency